tmol/score/omega/params.py

```diff
@@ -0,0 +1,36 @@
+import attr
+import cattr
+
+import torch
+
+from tmol.types.torch import Tensor
+from tmol.types.tensor import TensorGroup
+from tmol.types.array import NDArray
+from tmol.types.attrs import ValidateAttrs
+from tmol.types.functional import validate_args
+
+from tmol.database.scoring.omega import OmegaDatabase
+
+
+@attr.s(auto_attribs=True, slots=True, frozen=True)
+class OmegaGlobalParams(TensorGroup):
+    K: Tensor[torch.float32][...]
+
+    @classmethod
+    @validate_args
+    def from_database(cls, omega_database: OmegaDatabase, device: torch.device):
+        # Convert float entries into 1-d tensors
+        def at_least_1d(t):
+            if t.dim() == 0:
+                return t.expand((1,))
+            else:
+                return t
+
+        global_params = OmegaGlobalParams(
+            **{
+                n: at_least_1d(torch.tensor(v, device=device))
+                for n, v in cattr.unstructure(omega_database.global_parameters).items()
+            }
+        )
+
+        return global_params
```

Comment:
This is basically just packing the OmegaDatabase into tensors so they're available on the GPU. Does this look right?

From here, the OmegaWholePoseScoringModule labels these as parameters with the line: 

`self.global_params = _p(torch.stack(_t([global_params.K]), dim=1))`

which concatenates them all into a single tensor? Is this more complicated than it needs to be?

Reply:
Doesn't the value K need to be converted to a tensor somewhere?

Reply:
I'm inclined to leave this, so that it can serve as a more general example of how to include parameters for anyone wishing to write a score term in the future. If you'd prefer it be simplified, I can change it.